projects/ngx-vega/src/lib/vega.component.scss

```diff
@@ -5,6 +5,6 @@
 
   & > div {
     width: 100%;
-    height: 100%;
+    height: 99%;
```

Comment:
Did you check if this broke the other visualizations? Seems like it might break stuff.